src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1761,75 +1596,44 @@ void calculateEgoNet(string graphID, string partitionID, int serverPort, Jasmine
         bcopy((char *)server->h_addr, (char *)&serv_addr.sin_addr.s_addr, server->h_length);
         serv_addr.sin_port = htons(port);
         if (Utils::connect_wrapper(sockfd, (struct sockaddr *)&serv_addr, sizeof(serv_addr)) < 0) {
-            std::cout << "ERROR connecting" << std::endl;
+            instance_logger.error("ERROR connecting to port " + std::to_string(port));
             // TODO::exit
```

Comment:
Shall we remove `// TODO::exit`?

Reply:
Removed all occurrences of `// TODO::exit`

Reply:
Ok